docs/src/main/tut/docs/index.md

```diff
@@ -5,3 +5,190 @@ section: "docs"
+libraryDependencies ++= Seq(
+  "com.sksamuel.elastic4s" %% "elastic4s-core" % elastic4sVersion,
+  // for the tcp client
+  "com.sksamuel.elastic4s" %% "elastic4s-tcp" % elastic4sVersion,
+  
+  // for the http client
+  "com.sksamuel.elastic4s" %% "elastic4s-http" % elastic4sVersion,
+  
+  // if you want to use reactive streams
+  "com.sksamuel.elastic4s" %% "elastic4s-streams" % elastic4sVersion,
+  
+  // a json library
+  "com.sksamuel.elastic4s" %% "elastic4s-jackson" % elastic4sVersion,
+  "com.sksamuel.elastic4s" %% "elastic4s-play-json" % elastic4sVersion,
+  "com.sksamuel.elastic4s" %% "elastic4s-circe" % elastic4sVersion,
+  "com.sksamuel.elastic4s" %% "elastic4s-json4s" % elastic4sVersion,
+  
+  // testing
+  "com.sksamuel.elastic4s" %% "elastic4s-testkit" % elastic4sVersion % "test",
+  "com.sksamuel.elastic4s" %% "elastic4s-embedded" % elastic4sVersion % "test"
+)
+```
+
+## Your first request
+
+```tut:invisible
+import java.io.IOException
+import java.nio.file._
+import java.nio.file.attribute.BasicFileAttributes
+
+// setup a home directory
+val clusterName: String = "getting-started-with-elastic4s"
+val homePath: Path = Files.createTempDirectory(clusterName)
+```
+
+All examples use a local node for demonstration purpose. You can start an embedded node with
+
+```tut:silent
+import com.sksamuel.elastic4s.embedded.LocalNode
+
+import scala.concurrent.Await
+import scala.concurrent.duration._
+
+val localNode = LocalNode(clusterName, homePath.toAbsolutePath.toString)
+val client = localNode.elastic4sclient()
+```
+
+You can import the DSL and a default execution context with and execute a simple cluster state request with
+
+```tut:book
+import com.sksamuel.elastic4s.ElasticDsl._
+
+// await is a helper method to make this operation synchronous instead of async
+// You would normally avoid doing this in a real program as it will block your thread
+val response = client.execute {
+  clusterState()
+}.await
+
+response.getClusterName().value
+
+localNode.close()
+```
+
+## Example Application
+
+An example is worth 1000 characters so here is a quick example of how to connect to a node with a client, create and
+index and index a one field document. Then we will search for that document using a simple text query.
+
+For this example we will use the `elastic4s-circe` json serializer.
+
+
+```tut:silent
+import com.sksamuel.elastic4s.{ElasticsearchClientUri, TcpClient}
+import com.sksamuel.elastic4s.ElasticDsl._
+import org.elasticsearch.action.support.WriteRequest.RefreshPolicy
+import org.elasticsearch.common.settings.Settings
+
+// circe
+import com.sksamuel.elastic4s.circe._
+import io.circe.generic.auto._ 
+
+case class Artist(name: String)
+
+object ArtistIndex extends App {
+
+  // Here we create an instance of the TCP client
+  val client = TcpClient.transport(
+    Settings.builder().put("cluster.name", clusterName).build(),
+    ElasticsearchClientUri(s"elasticsearch://${localNode.ipAndPort}")
```

Comment:
I wonder why this doesn't work. I don't want to use the `localNode.elastic4sClient()` because
this examples should show how to start a TcpClient.